stix_shifter_modules/guardium/stix_translation/data_mapper.py

```diff
@@ -0,0 +1,13 @@
+from os import path
+import json
+from stix_shifter_utils.stix_translation.src.utils.exceptions import DataMappingException
+from stix_shifter_utils.modules.base.stix_translation.base_data_mapper import BaseDataMapper
+
+class DataMapper(BaseDataMapper):
+
+    def __init__(self, options, dialect=None):
+        super().__init__(dialect)
+        mapping_json = options['mapping'] if 'mapping' in options else {}
+        basepath = path.dirname(__file__)
+        self.map_data = mapping_json or self.fetch_mapping(basepath)
+
```

Comment:
data_mapper.py doesn't exist anymore in the latest develop and also its not used in the translation module as well.

Reply:
File was removed.